torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -741,6 +741,10 @@ absl::flat_hash_map<std::string, absl::variant<int>> ConvertDictToMap(
   return map;
 }
 
+// Override some upstream torch::lazy env vars for better performance.
+// Upstream lazy env vars defined in torch/csrc/lazy/core/config.h.
+void SetDefaultLazyEnvVars() { FLAGS_torch_lazy_handle_special_scalars = true; }
```

Comment:
Thanks @ymwangg , do we know if it's always safe/better to use constants for special scalars?

Reply:
This has been what pytorch/xla has been doing, it is just when we migrate to LTC, we also need to enable a FLAG to make this behavior consistent.

Reply:
I guess the right thing to do here is to do `FLAGS_torch_lazy_handle_special_scalars = xla::sys_util::GetEnvBool("XLA_NO_SPECIAL_SCALARS", false)`

given in 
https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor.cpp#L257

we also have an env var to control this behavior

Reply:
I will merge this one to unblock aws folks, but submit a new one to fix it.

Reply:
Thanks for the catch! You are right I overlooked XLA_NO_SPECIAL_SCALARS. I can fix it here or in future PR, whichever works best for you.

Reply:
If you don't mind, can you update in this pr?